lighthouse-cli/test/cli/index-test.js

```diff
@@ -3,6 +3,8 @@
  * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
  * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
  */
+// eslint-disable-next-line spaced-comment
+/// <reference types="jest" />
```

Comment:
Interesting, I've never had to do this in my other projects with jest, but maybe that's because of some other explicit includes in our `tsconfig.json`?

Reply:
If I change the file to `.ts` the explicit type include is not necessary.

Reply:
Awesome, that worked.